hbase-thrift/src/main/resources/hbase-webapps/thrift/thrift.jsp

```diff
@@ -31,9 +31,11 @@ String serverType = (String)getServletContext().getAttribute("hbase.thrift.serve
 long startcode = conf.getLong("startcode", System.currentTimeMillis());
 String listenPort = conf.get("hbase.regionserver.thrift.port", "9090");
 ImplType implType = ImplType.getServerImpl(conf);
-String framed = implType.isAlwaysFramed()
-    ? "true" : conf.get("hbase.regionserver.thrift.framed", "false");
-String compact = conf.get("hbase.regionserver.thrift.compact", "false");
+
+String transport = implType.isAlwaysFramed() ||conf.getBoolean("hbase.regionserver.thrift.framed", false) ? "Framed" : "Standard";
```

Comment:
Couple of nits:

- Ternary operator's boolean expression should be parenthesized, as per the Java style guide that HBase follows. Also a space after '||'. https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-136091.html
-  Can you please take care of max line width (100 chars). Looks like checkstyle does not run in jsp files. GOod to fix it before committing.

Reply:
+1 to the suggestions, rest looks good

Reply:
I corrected these, I hope it looks good now :)